### PR TITLE
Use jupyterlabs sim4life in the e2e

### PR DESCRIPTION
**Pull request by colinRawlings**
# 🐛 Switch e2e to jupyterlab sim4life 

## What do these changes do?

Improve the focus of the `jupyters` e2e test by:
- using the `sim4life jupyter` which uses the dynamic sidecar, jupyterlab3 and sim4life
- check that running basic sim4life functionality succeeds (parallel plates tutorial including solve and postpro) 

Also:
- Increase the time for shutdown to avoid failures with services that use the dynamic sidecar (e.g. `Sim4Life`)

## How to test

Run the e2e test following Odei's instructions at the top of: `osparc-simcore/tests/e2e/tutorials/jupyterlabs.js`

### CODE REVIEW

File: tests/e2e/tutorials/jupyterlabs.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+      workbenchData["studyId"],
+      [workbenchData["nodeIds"][1], workbenchData["nodeIds"][2]],
+      startTimeout,
+      false
+    );
+
+    await tutorial.waitFor(2000);
+
+    for (let j = 1; j < 3; j++) {
+      // open jupyterNB
```
Comment (odeimaiz) [inline tests/e2e/tutorials/jupyterlabs.js:37]:
```suggestion
      // open JLab
```

File: tests/e2e/tutorials/jupyterlabs.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+      await tutorial.openNode(j);
+      await tutorial.waitFor(35000);
+
+
+      // Run the jlab nbook
+      let iframeHandles2 = await tutorial.getIframe();
```
Comment (odeimaiz) [inline tests/e2e/tutorials/jupyterlabs.js:43]: all ```let```s can now be consts

File: tests/e2e/tutorials/tutorialBase.js [hunk 3]
```diff
@@ -253,7 +253,12 @@ class TutorialBase {
     while ((new Date().getTime()) - start < timeout) {
       for (let i = nodeIds.length - 1; i >= 0; i--) {
         const nodeId = nodeIds[i];
-        if (await utils.isServiceReady(this.__page, studyId, nodeId) && await utils.isServiceConnected(this.__page, studyId, nodeId)) {
+        let isLoaded = await utils.isServiceReady(this.__page, studyId, nodeId);
+        if(waitForConnected) {
```
Comment (odeimaiz) [inline tests/e2e/tutorials/tutorialBase.js:257]:
```suggestion
        if (waitForConnected) {
```

**Review by odeimaiz — APPROVED** [review]

**Review by KZzizzle — APPROVED** [review]
:muscle:

**Review by KZzizzle — APPROVED** [review]
second time's the charm

**Review by pcrespov — COMMENTED** [review]
👍

**Review by sanderegg — APPROVED** [review]
very nice! thanks!

**Pull request merged**